src/ExpressionEvaluator/Core/Source/ExpressionCompiler/PseudoVariableUtilities.cs

```diff
@@ -54,5 +54,14 @@ internal static DkmClrCompilationResultFlags GetLocalResultFlags(this Alias alia
                     return DkmClrCompilationResultFlags.None;
             }
         }
+
+        internal static bool IsReturnValue0(this Alias alias)
```

Comment:
The return values are 1-based.  You'll either need to filter out "$ReturnValue" or "$ReturnValue1"